crates/net/network/src/peers/manager.rs

```diff
@@ -856,6 +866,7 @@ impl Default for ConnectionInfo {
             max_outbound: DEFAULT_MAX_PEERS_OUTBOUND,
             max_inbound: DEFAULT_MAX_PEERS_INBOUND,
             max_concurrent_outbound_dials: DEFAULT_MAX_CONCURRENT_DIALS,
+            max_total_peers: 0,
```

Comment:
this we don't need because we already have this via max_inbound

Reply:
> this we don't need because we already have this via max_inbound

i'm having some issue right now with reth and rust analyzer, rust analyzer detect crate conflit so i don't know what's going on exactly 

i will return to this when i finish my rust analyzer problem, it say it have build failure  with libffi-sys